src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a = 0, b = 0 }) {
+  const result = a + b;
+
+  return (
+    <p>{`Sum of ${a} and ${b} is ${result}`}</p>
```

Comment:
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```
You can also calculate it without variable just inside template string